services/src/main/java/org/exoplatform/news/rest/NewsTargetingRestResourcesV1.java

```diff
@@ -202,4 +210,38 @@ public Response undoDeleteTarget(@Context HttpServletRequest request,
     }
   }
 
+  @POST
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("users")
+  @ApiOperation(
+          value = "Create a new target.",
+          httpMethod = "POST",
+          response = Response.class
+  )
+  @ApiResponses(
+          value = {
+                  @ApiResponse(code = HTTPStatus.BAD_REQUEST, message = "Invalid query input"),
+                  @ApiResponse(code = HTTPStatus.FORBIDDEN, message = "Forbidden operation"),
+                  @ApiResponse(code = HTTPStatus.UNAUTHORIZED, message = "User not authorized to create news target"),
+                  @ApiResponse(code = HTTPStatus.CONFLICT, message = "Conflict operation") }
+  )
+  public Response createNewsTarget(@Context HttpServletRequest request,
+                                       @ApiParam(value = "News target to create", required = true) Metadata newsTarget) {
+    long userIdentityId = NewsUtils.getCurrentUserIdentityId(identityManager);
+    try {
+      Metadata addedNewsTarget = newsTargetingService.createMetadata(newsTarget, userIdentityId);
```

Comment:
I think that you should juste send NewsTargetingEntity as a parameter, then instanciate the corresponding metadata to be sent to metadata Service